src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -62,6 +81,17 @@ main() {
     echo "============================================================="
 
     filtered_inputs=""
+
+    # initiate log file and start forwarding it to the job monitor 
+    touch $LOG_NAME
+    if [[ $debug ]]; then
+      (while true; do truncate --size="<5G" $LOG_NAME; sleep 10; done) &
```

Comment:
5G is pretty huge, I think the log size is capped at 2MB or 100k lines.

Reply:
https://github.com/dnanexus/nucleus/blob/4586131b460dd059bfe6a6c187698c81db2b0224/execserver/scripts/relay_job_logging.py#L179

I saw we are using 2MB or 100 lines as the limit. Will adopt those configs. Thanks @mhrvol !

Reply:
Refactored.